src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionVisibilityOptions.tag

```diff
@@ -12,9 +12,10 @@
 <c:set var="FEEDBACK_STUDENTS"><%=FeedbackParticipantType.STUDENTS.name()%></c:set>
 <c:set var="FEEDBACK_INSTRUCTORS"><%=FeedbackParticipantType.INSTRUCTORS.name()%></c:set>
 
-<div class="col-sm-12 margin-bottom-15px padding-15px background-color-light-green">
+<div class = "col-sm-12 margin-bottom-15px padding-15px <%= fqForm.isQuestionHasResponses() ? "alert alert-danger" : "background-color-light-green" %>">
     <div class="margin-bottom-7px">
-        <b class="visibility-title">Visibility</b> (Who can see the responses?)
+        <h4><%= fqForm.isQuestionHasResponses() ? "Changing the visibility after collecting responses is not recommended." : "" %></h4>
+        <b class=<%= fqForm.isQuestionHasResponses() ? "" : "visibility-title" %>>Visibility</b> (Who can see the responses?)
```

Comment:
Is there a reason why class `visibility-title` was dropped for questions with responses? What does it look like with the class?